tests/test_expectation_maximization.py

```diff
@@ -99,11 +115,37 @@ def test_generate_slices(test_ir, n_particles, n_pix):
     rots = test_ir.grid_SO3_uniform(n_particles)
     xy_plane = test_ir.generate_xy_plane(n_pix)
 
-    slices, xyz_rotated = test_ir.generate_slices(map_3d, xy_plane, n_pix, rots)
+    slices, xyz_rotated_planes = test_ir.generate_slices(map_3d, xy_plane, n_pix, rots)
 
-    assert xy_plane.shape == (n_pix**2, 3)
     assert slices.shape == (n_particles, n_pix, n_pix)
-    assert xyz_rotated.shape == (n_pix**2, 3)
+    assert xyz_rotated_planes.shape == (n_particles, 3, n_pix**2)
+
+    rot_test_map = np.array(
+        [
+            [[0, 0, 0], [0, 0, 0], [0, 0, 0]],
+            [[1, 1, 1], [1, 1, 1], [1, 1, 1]],
+            [[0, 0, 0], [0, 0, 0], [0, 0, 0]],
+        ]
+    )
```

Comment:
your map should be nxnxn where n is even. Here n is 3. Keep it small for readability. Make n=4 or 2.